app/jobs/faq_csv_import_job.rb

```diff
@@ -3,38 +3,35 @@
 # faq_csv = File.read("../../Downloads/faq.csv")
 # result = FaqCsvImportJob.perform_now(File.read("../../Downloads/faq.csv"))
 # File.open('/tmp/test_faq.yml', 'w') {|f| f.write result.to_yaml }
-class FaqCsvImportJob < ApplicationJob
-  def perform(faq_csv_content, write: false)
+class FaqCsvImportJob
+  def self.perform_now(faq_csv_content, write: false)
     [:en, :es].each do |locale|
       filename = "config/locales/#{locale}.yml"
-      puts 'parse_results'
-      # puts self.class.parse(faq_csv_content, locale).to_json
-      new_translations = self.class.updated_translations(
-        YAML.safe_load(File.read("config/locales/#{locale}.yml")).with_indifferent_access,
+      translation_data = YAML.load_file("config/locales/#{locale}.yml")
```

Comment:
maybe we should use the old YAML method? can check

Reply:
cool